lib/rest-api.php

```diff
@@ -85,6 +85,10 @@ function gutenberg_register_rest_widget_areas() {
  * @since 6.5.0
  */
 function gutenberg_register_rest_block_directory() {
+	if ( ! gutenberg_is_experiment_enabled( 'gutenberg-block-directory' ) ) {
+		return;
+	}
+
```

Comment:
I think these are the changes that feel less in line with the "pluginisation" of the block directory, but I understand that the primary goal of the PR is to avoid the bundling of client-side assets, since for the server we have leeway in how we merge things into Core.

Reply:
Yes, it isn't a true plugin, it's more a way to avoid exposing unused code. All routes are exposed in the code generated in HTML.

Reply:
Fixed in 3f3d015. Tested with `npm run plugin-build`:

<img width="678" alt="Screen Shot 2019-10-28 at 10 59 19" src="https://user-images.githubusercontent.com/699132/67669485-48887080-f972-11e9-96e1-4efd87242dd1.png">